distribution/scripts/ballerina/data_service/main.bal

```diff
@@ -0,0 +1,126 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+
+configurable string dbUser = ?;
+configurable string dbPassword = ?;
+configurable string dbHost = ?;
+configurable int dbPort = ?;
+configurable string dbName = ?;
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+type Order record {|
+    string symbol;
+    string buyerID;
+    float price;
+    int volume;
+|};
+
+type DBOrder record {|
+    int id;
+    string symbol;
+    string buyerID;
+    float price;
+    int volume;
+|};
+
+type Invoice record {|
+    Order[] 'order;
+|};
+
+type Request record {|
+    string size;
+    Invoice payload;
+|};
+
+listener http:Listener securedEP = new (9090,
+    secureSocket = {
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+final mysql:Client dbClient = check new (
+    host = dbHost,
+    user = dbUser,
+    password = dbPassword,
+    port = dbPort,
+    database = dbName
+);
+
+service /'orders on securedEP {
+
+    isolated resource function post .(Request request) returns int|error? {
```

Comment:
Do we have to explicitly say this function as `isolated`? Check other places as well